RxWebKit/Sources/RxWKNavigationDelegateProxy.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  RxWKNavigationDelegateProxy.swift
+//  RxWebKit
+//
+//  Created by Bob Obi on 23.10.17.
+//  Copyright © 2017 RxSwift Community. All rights reserved.
+//
+
+import WebKit
+#if !RX_NO_MODULE
+    import RxSwift
+    import RxCocoa
+#endif
+
+public typealias RxWKNavigationDelegate = DelegateProxy<WKWebView, WKNavigationDelegate> 
+
+open class RxWKNavigationDelegateProxy: RxWKNavigationDelegate, DelegateProxyType, WKNavigationDelegate {
+    
+    /// Type of parent object
+    /// must be WKWebView!
```

Comment:
You can probably get rid of the "must be WKWebView", it's obvious based on the type of the variable

Reply:
Actually I thought of it and again I decided to keep because it will avoid you from referencing self.webView within the subscription if you were to access the webView for some updates. I might be wrong though..

Reply:
I'm not sure how it would matter in this case? 
My nit here is that you write "must be X" about `var something: X` ... the latter already defines the type, so the comment isn't really necessary here

Reply:
My bad that was a wrong comment. Sorry I was thinking of something totally different.

Reply:
No worries 👍